src/pages/debrief/debrief.analytics.effects.ts

```diff
@@ -22,33 +25,30 @@ export class DebriefAnalyticsEffects {
     private actions$: Actions,
     private store$: Store<StoreModel>,
   ) {
-    this.analytics.initialiseAnalytics()
-          .then(() => {})
-          .catch(() => {
-            console.log('error initialising analytics');
```

Comment:
I'm happy getting rid of these logs. However we will be hiding a problem which is that we aren't using the analytics mock in the tests. Running the tests on develop we get this log 94 times. 

Doing some quick trial we need to add the following to our test classes where analytics could be called from (Pages, Components, Effects etc) 

`{ provide: AnalyticsProvider, useClass: AnalyticsProviderMock },`